libs/ui-watt/src/lib/components/drawer/watt-drawer-actions-content.component.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'watt-drawer-actions-content',
+  template: `<ng-content></ng-content>`,
+  styles: [
+    `
+      :host {
+        margin-left: var(--watt-space-ml);
+      }
+    `,
+  ],
+})
+export class WattDrawerActionsContentComponent {}
```

Comment:
suggestion: looks like the component is used primarily with a heading, so we can also call it `watt-drawer-heading`

Reply:
Resolved in https://github.com/Energinet-DataHub/greenforce-frontend/pull/1497/commits/0c5088e6dfc35ef80e278709b0dce6a24774bf30